bridges/otelslog/handler.go

```diff
@@ -172,6 +188,16 @@ func (h *Handler) convertRecord(r slog.Record) log.Record {
 	const sevOffset = slog.Level(log.SeverityDebug) - slog.LevelDebug
 	record.SetSeverity(log.Severity(r.Level + sevOffset))
 
+	if h.source {
+		fs := runtime.CallersFrames([]uintptr{r.PC})
+		f, _ := fs.Next()
+		record.AddAttributes(log.Map("source",
```

Comment:
Should we use the following semantic conventions?
https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/code.md

Reply:
Totally! So we should have 3 separate string attributes instead of one `source` map attribute with 3 attributes:
- `code.function`
- `code.filepath`
- `code.lineno`

Reply:
Sounds good! But is there some shortcut in converting the `attribute.KeyValue` into a `log.KeyValue` or is it just:
```
record.AddAttributes(
			log.String(string(semconv.CodeFunctionKey), semconv.CodeFunction(f.Function).Value.AsString()),
			log.String(string(semconv.FilePathKey), semconv.FilePath(f.File).Value.AsString()),
			log.Int(string(semconv.CodeLineNumberKey), int(semconv.CodeLineNumber(f.Line).Value.AsInt64())),
		)
```

Reply:
There is no shortcut. Sorry about that. Also please just use the Key constantants. E.g. 

```go
log.String(string(semconv.CodeFunctionKey), f.Function)
```

Reply:
updated using `semconv.CodeFilepathKey`, `semconv.CodeFunctionKey`, and `semconv.CodeLineNumberKey`